shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "backend/protobuf/proto_query_response_factory.hpp"
+#include "backend/protobuf/permissions.hpp"
+#include "backend/protobuf/query_responses/proto_block_query_response.hpp"
+#include "backend/protobuf/query_responses/proto_query_response.hpp"
+
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createAccountAssetResponse(
+    std::vector<std::shared_ptr<shared_model::interface::AccountAsset>> assets,
```

Comment:
Why not `vector<unique_ptr>`, so that it is possible to move the assets to query response? Same in `createTransactionsResponse`

Reply:
`unique_ptr` seems not that handy to use, since most of the methods in iroha works with `shared_ptr`